### PR TITLE
SSH and SCP

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by jovirkku** [comment]
Help_Articles:
    SSH_and_SCP:
        SSH_and_SCP_Linux
        SSH_and_SCP_Mac
        SSH_and_SCP_Windows

**Review by jpwalden — APPROVED** [review]
Looks good to me and good idea to structure these documents this way.

File: Support/Help_Articles/SSH_and_SCP/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: SSH and SCP
+permalink: Support/Help_Articles/SSH_and_SCP/
```
Comment (vigejolla) [inline Support/Help_Articles/SSH_and_SCP/README.md:3]: missing parent

File: Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+---
+title: SSH and SCP (Linux)
+permalink: Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/
+parent: SSH_and_SCP
```
Comment (vigejolla) [inline Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md:4]: parent should be the title and not directory name, i.e. `SSH and SCP`

Comment (vigejolla) [reply Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md:4]: Also needs `grand_parent: Help Articles`

Comment (vigejolla) [reply Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md:4]: The above comments apply to all three child documents

**Review by vigejolla — CHANGES_REQUESTED** [review]
I like the idea, but some fixes are needed. Also, I think we should consider combining some of the already existing pages under a common page, e.g. All "Setup * Account" pages could be under a single "Account Setup" page. That's outside of the scope of this task, but just saying :)

File: Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md (hunk 2 above)
Comment (jovirkku) [reply Support/Help_Articles/SSH_and_SCP/SSH_and_SCP_Linux/README.md]: Parents and Grandparents fixed.

File: Support/Help_Articles/SSH_and_SCP/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/SSH_and_SCP/README.md:3]: Added

**Comment by jovirkku** [comment]
I believe this is ready now.

**Comment by jovirkku** [comment]
A hyperlink at the end of the Mac document was still fixed, thanks to PR #192.

**Review by vigejolla — APPROVED** [review]
LGTM, but if you don't mind I'll merge this together with PR #192.

**Pull request merged**